indico/modules/events/registration/util.py

```diff
@@ -185,3 +185,14 @@ def generate_csv_from_registrations(registrations, regform_items, special_items)
         writer.writerow(registration_dict)
     buf.seek(0)
     return buf
+
+
+def get_registrations_with_tickets(user, event):
+    return Registration.find(Registration.user == user,
+                             RegistrationForm.event_id == event.id,
+                             ~Registration.is_deleted,
+                             ~RegistrationForm.is_deleted,
+                             Registration.state == RegistrationState.complete,
+                             RegistrationForm.tickets_enabled,
+                             RegistrationForm.ticket_on_event_page,
+                             _join=Registration.registration_form).all()
```

Comment:
return Registration.find(Registration.user == user,
                             Registration.state == RegistrationState.complete,
                             RegistrationForm.event_id == event.id,
                             RegistrationForm.tickets_enabled,
                             RegistrationForm.ticket_on_event_page,
                             ~RegistrationForm.is_deleted,
                             ~Registration.is_deleted,

Sorting the lines like this  seems more readable to me